src/Uploader/useUploader.ts

```diff
@@ -77,11 +77,31 @@ export default function useUploader({
     const bytesToMb = maxFileSize / 1024 / 1024;
     const limitSize = `${bytesToMb.toFixed()} MB`;
```

Comment:
This code can be moved in to the appropriate case as it's not needed for all statements.